### DISCUSSION TITLE
Blog "The end (of the first semester) is nigh" - PeerReviews

### DISCUSSION THREAD

**Discussion by qvalentin** [FileFighter/filefighter.github.io#50] [General]
Leave your feedback about our new blog post below :D

**Comment by Niklas-23** [comment]
Hey FileFighter Team,

the first screencast from your frontend looks really good. It seems like your project is going very well. In addition, the documentation of the other use cases is very detailed. Have you already implemented this use cases and could you show a screencast?

You have already written detailed .feature files in the documentation of the use cases. It would be interesting to see if you can already run the tests?

Greetings,
the Live-Poll Team

**Comment by open-schnick** [comment]
> Hey FileFighter Team,
> 
> the first screencast from your frontend looks really good. It seems like your project is going very well. In addition, the documentation of the other use cases is very detailed. Have you already implemented this use cases and could you show a screencast?
> 
> You have already written detailed .feature files in the documentation of the use cases. It would be interesting to see if you can already run the tests?
> 
> Greetings,
> the Live-Poll Team

Hello Live-Poll,
Thank you for your feedback. 
We added a screencast of our running tests in our cucumber tests [blog](https://filefighter.github.io/tech/2020/10/31/cucumber-testing.html).
Generally speaking every step file, that is not commented out is already implemented, and running. Same goes for the use cases. 
Greetings FileFighterTeam

**Comment by Honrix** [comment]
Hi FileFighters!

Thank you for your blog post and the screen cast. 

What we liked:
- Screencast shows progress
- is embedded
- SAD is very detailed
- design is simple and compact
- every has a direct link (e.g. SAD)

Things you could improve:
- Maybe you could give a short explanation why you chose Spring and ReactJS. If you’ve already explained it in another blog you could link to that

Good luck on your project and keep grinding!

Best wishes for Filefighters,
Plandora-Team
